src/sentry/static/sentry/app/components/shortId.tsx

```diff
@@ -5,12 +5,18 @@ import isPropValid from '@emotion/is-prop-valid';
 
 import AutoSelectText from 'app/components/autoSelectText';
 
-export default class ShortId extends React.Component {
+type Props = {
+  shortId: string;
+  avatar?: React.ReactNode;
+};
+
+export default class ShortId extends React.Component<Props> {
   static propTypes = {
-    shortId: PropTypes.string,
+    shortId: PropTypes.string.isRequired,
```

Comment:
@matejminar hmmm...why are we making it required?

Reply:
I think this component does not make sense without shortId prop, so I made it required. I scanned the entire codebase and it should not be a problem. I wanted to have the component as strictly typed as possible by default and adjust the type in the future to be more relaxed if needs to be. Do you think we can keep it like that, or should I revert it?

Reply:
I've confirmed the props usage. Please keep it 👍